src/static/containers/User/index.js

```diff
@@ -1,39 +1,81 @@
-import React from 'react';
+import React, { Component } from 'react';
+import { push } from 'react-router-redux';
 import { connect } from 'react-redux';
 import { bindActionCreators } from 'redux';
 import PropTypes from 'prop-types';
 
 import * as actionCreators from '../../actions/users';
 
-const ShortUserView = ({ user }) => (
-    <div className="panel panel-default card">
-        <div className="panel-body">
-            <div className="col-lg-10">
-                <h2>{user.username}</h2>
-                <p>{user.email}</p>
+class ShortUserView extends Component {
+    deleteUser = () => {
+        const { actions, token, user } = this.props;
+        actions.deleteUser(token, user.username);
+    };
+
+    goToEditUser = () => {
+        const { username } = this.props.user;
+        this.props.dispatch(push(`/zw-admin/user-edit/${username}`));
+    }
+
+    render = () => {
+        const { user, admin } = this.props;
+        const adminBtns = [];
+        if (admin) {
+            const { username } = user;
+            adminBtns.push(<i key={`delete-btn-${username}`}
+                className="cursor fa fa-times"
+                onClick={this.deleteUser}
+            />);
+            adminBtns.push(<i key={`edit-btn-${username}`}
+                className="cursor fa fa-pencil"
+                onClick={this.goToEditUser}
+            />);
+        }
+        return (
+            <div className="panel panel-default user">
+                <div className="panel-body">
+                    <div className="row">
+                        {adminBtns}
+                    </div>
+                    <div className="col-lg-10">
+                        <h2>{user.username}</h2>
+                        <p>{user.email}</p>
+                    </div>
+                </div>
             </div>
-        </div>
-    </div>
-);
+        );
+    };
+}
+
+ShortUserView.defaultProps = {
+    admin: false
+};
 
 ShortUserView.propTypes = {
+    admin: PropTypes.bool.isRequired,
```

Comment:
admin may be store in the user?
Or is this something different than the rights the user have?

Reply:
The user props refers to the displayed user, the admin props refers to the users viewing the card.